citizenvoice/survey_design/views.py

```diff
@@ -16,7 +16,11 @@
 
 def index(request):
     form = UserCreationForm()
-    return render(request, 'survey_design/index.html')
+    context = {
+        'surveys': SurveyViewSet.GetSurveyByDesigner(request.user.id)
+    }
+    #TODO: Get all available surveys, do not filter by designer
```

Comment:
Does the TODO mean that this should be changed to show all surveys? The right API command would then be
`SurveyViewSet.get_queryset()` to return all surveys.

Reply:
We need a new API command that filters surveys based on today's date, publication and expiration dates, and returns the ones available at the time. I thought we can have it in the next PR, but I am also fine with writing it if you think it is the way to go